---
title: "Lavrov Lab - Research"
layout: textlay
excerpt: "Lavrov Lab -- Research"
sitemap: false
permalink: /research/
---

# Research

Our group studies evolution and phylogeny of non-bilaterian animals -- *i.e.*, animals without bilateral symmetry -- including Porifera (sponges), Cnidaria (corals, jellyfishes), and Ctenophora (comb jellies) (Fig. 1).
While morphologically simple and moderately speciose, these groups harbor a large portion of animal genomic diversity that we are only now beginning to explore.
Our research can be divided into three major areas: 1) comparative genomics, the comparison of biological information derived from whole-genome sequences, 2) phylogenetics, the study of evolutionary relationships among organisms (aka Tree of Life), and 3) mito-nuclear coevolution – exploration of interdependence between changes in nuclear and mitochondrial genomes.
Our research is highly integrative and includes field sampling, wet-lab work, and bioinformatic analysis.
We have also been involved in multiple collaborations on the national and international levels.
Ongoing projects include:

![]({{ site.url }}{{ site.baseurl }}/images/respic/sponge.jpg){: style="width: 300px; float: right; border: 10px"}

## Phylomitogenomics and comparative mitochondrial genomics of non-bilaterian animals.
Mitochondrial DNA of bilaterian animals is typically a small, circular-mapping molecule that encodes 37 tightly packed genes.
MtDNA of the choanoflagellate _Monosiga brevicollis_, the closest unicellular out-group, is four times larger and contains 1.5 times as many genes.
We investigated this remarkable transition in mtDNA evolutionby focusing on non-bilaterian animals (phyla Placozoa, Porifera, Ctenophora, and Cnidaria).
Check out [our review](https://doi.org/10.1093/gbe/evw195) on the topic to learn more.
We also use coding sequences from mt-genomes to infer phylogeny within and among non-bilaterian animals.
In particular, we created and maintain a [website](https://lavrovlab.github.io/Demosponge-phylogeny/) on phylomitogenomics of demosponges.

## Function and evolution of mt-mutS in octocorals:  
Typically, animal mitochondrial DNA (mtDNA) has a higher substitution rate than nuclear DNA, with the accumulation of mtDNA mutations being one of the hallmarks of aging.
This discrepancy in the rates of evolution is partially due to the supposed lack of mismatch repair activities in the mitochondria. Octocorals – a group of cnidarians – have a reduced rate of mitochondrial evolution and encode a mismatch repair-like protein (mt-MutS) in their mtDNA.
Previous analyses suggested that this enzyme was acquired from a virus and has been universally retained among octocoral taxa.
However, whether it is involved in mtDNA repair and is responsible for the lower rate of nucleotide substitution in octocoral mtDNA remains unknown.
In a collaboration with two groups in Europe we investigate structure, function, and evolution of mt-MutS.
Initial results of this research have been published in [Genome Biology and Evolution](https://doi.org/10.1093/gbe/evab191) and [DNA Repair](https://doi.org/10.1016/j.dnarep.2022.103273).

![]({{ site.url }}{{ site.baseurl }}/images/respic/protein.jpg){: style="width: 300px; float: left; border: 10px"}
## Evolution of mitochondrial proteome.  
In the 30 years since the first animal mtDNA has been published, we have learned a great deal about the evolution of mitochondrial genomes. However, because mtDNA encodes only a tiny fraction of mitochondrial proteins, the evolution of mitochondrial proteome remains largely unexplored. From the evolutionary viewpoint, this proteome represents an interesting system as it is made out of interacting components encoded by two genomes with different rates and modes of evolution and because some analogous molecules (e.g., components of the translation machinery) are encoded in both nuclear and mitochondrial DNA. We investigate both specific mitonuclear interactions, such co-evolution of nuclear-encoded aminoacyl-tRNA synthetases (aa-RS) and mitochondrial-encoded tRNAs (e.g., [Pett and Lavrov, 2015](https://doi.org/10.1093/gbe/evv124)) and overall mitochondrial proteome evolution ([Muthye and Lavrov, 2018](https://doi.org/10.1002/iub.1961}). We also develop tools for bioinformatic identification of mitochondrial proteins ([Muthye et al. 2020](https://doi.org/10.1016/j.mito.2020.01.001)).

## tRNA import in animal mitochondria
While mtDNA in most bilaterian animals encodes a complete set of tRNA genes needed for mitochondrial translation, three out of the four main lineages of nonbilaterian animals experienced significant losses of mt-tRNA genes. All mt-tRNA genes have been lost in Ctenophora, all but one or two mt-tRNA genes have been lost in Cnidarian, and a variable number of tRNA genes has been lost in sponges. Interestingly, while in most of these lineages, mt-tRNA loss has occurred early in evolution and the number of tRNA genes has been conserved thereafter, a group of haplosclerid sponges shows a large variation in the number of mt-tRNA genes among closely related species. We are taking advantage of this variation to identify components of mitochondrial tRNA import system need to import cy-tRNA replacements for lost mt-tRNAs. We then plan to investigate evolution of this system in other animal groups.
