---
title: "Lavrov Lab - Research"
layout: textlay
excerpt: "Lavrov Lab -- Research"
sitemap: false
permalink: /research/
---

# Research

Our main research interests are the evolution of major groups of animals and their genomes.  Our favorite animals are marine non-bilaterian taxa, including Porifera (sponges), Cnidaria (corals, jellyfishes), and Ctenophora (comb jellies). It is rarely appreciated that from the phylogenetic perspective each group of these animals forms at least an equal-status taxon to all bilaterian animal (from flatworms to humans) and thus should harbor a large portion of animal genomic diversity that we only now are beginning to explore.  Ongoing projects include:

![]({{ site.url }}{{ site.baseurl }}/images/respic/sponge.jpg){: style="width: 300px; float: right; border: 10px"}

## Comparative mitochondrial genomics of non-bilaterian animals.
Mitochondrial DNA of bilaterian animals is typically a small, circular-mapping molecule that encodes 37 tightly packed genes.  MtDNA of the choanoflagellate Monosiga brevicollis, the closest unicellular out-group, is four times larger and contains 1.5 times as many genes.  Comparative genomic studies can provide valuable insights into the mechanisms responsible for this remarkable transition in mtDNA evolution, but the scarcity of data from non-bilaterian animals (phyla Placozoa, Porifera, Ctenophora, and Cnidaria) severely limits these studies.  My group aims to sample and analyze mitochondrial genomes from all major groups of non-bilaterian animals.  Currently, we are working on the mitochondrial genomes of multiple sponges (see below), representatives of all major lineages in Cnidaria and Ctenophora.  Our analyses revealed a large extent of mitochondrial diversity in these groups.  We discovered such unusual features as mitochondrial introns, putative transposable elements, translational frameshifting, fragmented ribosomal RNA (rRNA) genes, transfer RNA (tRNA) gene recruitment, tRNA and mRNA editing.  We also found several cases of parallel evolution in mitochondrial genome organization in different groups.  Such cases can help to elucidate molecular and evolutionary forces responsible for unusual features in animal mtDNA, some of which have been implicated in human diseases. Here is our recent review. 

![]({{ site.url }}{{ site.baseurl }}/images/respic/protein.jpg){: style="width: 300px; float: left; border: 10px"}
## Evolution of mitochondrial proteome.
In the 30 years since the first animal mtDNA has been published, we have learned a great deal about the evolution of mitochondrial genomes. However, because mtDNA encodes only a tiny fraction of mitochondrial proteins, the evolution of mitochondrial proteome remains largely unexplored. From the evolutionary viewpoint, this proteome represents an interesting system as it is made out of interacting components encoded by two genomes with different rates and modes of evolution and because some analogous molecules (e.g., components of the translation machinery) are encoded in both nuclear and mitochondrial DNA. We began to explore the evolution of mitochondrial proteome by investigating the fate of nuclear-encoded aminoacyl-tRNA synthetases (aa-RS) in animals, where mitochondrial-encoded tRNAs are lost. We found that the loss of such tRNA genes in the sea anemone Nematostella and a ctenophore Mnemiopsis had lead to a loss of all but two nuclear-encoded aaRS in both organisms. In our recent study (Pett and Lavrov, 2015), we tested the universality of this observation by analyzing mitochondrial and nuclear genomes in a variety of animal species, including several lineages that lost some or all mitochondrial tRNAs. We are also interested in exploring the evolution of other elements of mitochondrial proteome in animals.
