---
title: "Lavrov Lab - Research"
layout: textlay
excerpt: "Lavrov Lab -- Research"
sitemap: false
permalink: /research/
---

# Research

![]({{ site.url }}{{ site.baseurl }}/images/respic/tree.jpg){: style="width: 300px; float: right; border: 10px"}
Our group studies molecular evolution in non-bilaterian animals -- *i.e.*, animals without bilateral symmetry -- including Porifera (sponges), Cnidaria (corals, jellyfishes), and Ctenophora (comb jellies) (Fig. 1).
While morphologically simple and moderately speciose, these groups harbor a large portion of animal genomic diversity that we are only now beginning to explore.
Our research can be divided into three major areas: 1) comparative mitochondrial genomics (mitogenomics), the study of organization and evolution of mitochondrial DNA, 2) phylogenetics, the study of evolutionary relationships among organisms (aka Tree of Life), and 3) mito-nuclear coevolution – exploration of interdependence between changes in nuclear and mitochondrial genomes.
Our research is highly integrative and includes field sampling, wet-lab work, and bioinformatic analysis.
We have also been involved in multiple collaborations on the national and international levels.  
Ongoing projects include:

![]({{ site.url }}{{ site.baseurl }}/images/respic/sponge.jpg){: style="width: 300px; float: right; border: 10px"}

## Comparative mitochondrial genomics of non-bilaterian animals.
Mitochondrial DNA of bilaterian animals is typically a small, circular-mapping molecule that encodes 37 tightly packed genes.  
MtDNA of the choanoflagellate Monosiga brevicollis, the closest unicellular out-group, is four times larger and contains 1.5 times as many genes.  
We investigated this remarkable transition in mtDNA evolutionby focusing on non-bilaterian animals (phyla Placozoa, Porifera, Ctenophora, and Cnidaria) severely limits these studies.  
Our analyses revealed a large extent of mitochondrial diversity in these groups.
We discovered such unusual features as mitochondrial introns, putative transposable elements, translational frameshifting, fragmented ribosomal RNA (rRNA) genes, transfer RNA (tRNA) gene recruitment, tRNA and mRNA editing.  
We also found several cases of parallel evolution in mitochondrial genome organization in different groups.  
Such cases can help to elucidate molecular and evolutionary forces responsible for unusual features in animal mtDNA, some of which have been implicated in human diseases.

![]({{ site.url }}{{ site.baseurl }}/images/respic/protein.jpg){: style="width: 300px; float: left; border: 10px"}
## Mito-nuclear coevolution.
Mitochondrial DNA of bilaterian animals is typically a small, uniform, circular-mapping molecule that encodes 37 tightly packed and fast-evolving genes.  
MtDNA of the choanoflagellate \textit{Monosiga brevicollis}, the closest sampled unicellular out-group to animals, is four times larger, contains 1.5 times as many genes, and displays much lower rates of sequence evolution.  
Comparative genomic studies can provide valuable insights into this remarkable transition in mtDNA organization, but scarcity of data from non-bilaterian animals had previously precluded such studies.
At the time we started this project, mitogenomic data were available from one class of one out of the four phyla of non-bilaterian animals.
Therefore, the goal of my group was to describe mtDNA diversity in the other taxa.
We were the first group to publish mitochondrial genomes from all four classes of sponges, four out of five classes of Cnidaria, and phylum Ctenophora.
In the process, we discovered a large extent of diversity in mt-genome organization among non-bilaterian animals, including circular vs. linear chromosomes, multipartite genomes, extra genes (e.g. *atp9*, *polB*, *tatC*), various number of encoded mitochondrial transfer RNAs (tRNAs) (0–25), at least seven different genetic codes, presence/absence of introns, tRNA and mRNA editing, fragmented ribosomal RNA genes, translational frameshifting, highly variable substitution rates, and a large range of genome sizes.
 This newly discovered diversity allowed a better understanding of the evolutionary plasticity and conservation of animal mtDNA and provided insights into the molecular and evolutionary mechanisms shaping mitochondrial genomes.


## Function and evolution of mt-mutS in octocorals}:  
Typically, animal mitochondrial DNA (mtDNA) has a higher substitution rate than nuclear DNA, with the accumulation of mtDNA mutations being one of the hallmarks of aging.
This discrepancy in the rates of evolution is partially due to the supposed lack of mismatch repair activities in the mitochondria. Octocorals – a group of cnidarians – have a reduced rate of mitochondrial evolution and encode a mismatch repair-like protein (mt-MutS) in their mtDNA.
Previous analyses suggested that this enzyme was acquired from a virus and has been universally retained among octocoral taxa.
However, whether it is involved in mtDNA repair and is responsible for the lower rate of nucleotide substitution in octocoral mtDNA remains unknown.
In a collaboration with two groups in Europe (those of \href{http://nmrbordeaux.org/}{Cameron Mackereth} at the Institut Européen de Chimie et Biologie (IECB) at the University of Bordeaux, France and \href{https://www.age.mpg.de/science/research-laboratories/stewart/}{Jim Stewart} at the Max Planck Institute for Biology of Aging in Cologne, Germany) we received funding from Human Frontier Science Program to investigate structure, function, and evolution of mt-MutS.
 As part of this project, our lab is conducting a phylogenetic comparative study of octocoral mtDNA to identify distinct mutation patterns among its lineages and correlate them with the changes in mt-MutS.
 We also calculate site- and taxa-specific evolutionary rates in mt-MutS to infer functional and structural constraints and to optimize the choice of the mt-MutS for transgenesis.
 Finally, we use phylogenetic analysis to elucidate the origin of \textit{mutS} in octocorals.


<!-- ## tRNA import in animal mitochondria
Although transfer RNA (mt-tRNA) genes constitute only 10\% of the human mt-genome, they account for more than half of 400+ characterized human mt mutations. 
These mutations are associated with a wide range of pathologies, from isolated organ-specific diseases such as myopathy or hearing loss to multi-system disorders involving central nervous, cardiovascular, musculoskeletal, and other systems.
Although there are no known cures for diseases caused by mt-tRNA mutations, import of cytosolic tRNAs into mitochondria has been suggested as a possible treatment.
Unfortunately, little is known about the mechanisms of mt-tRNA import in animals, in part because mtDNA in all animal model species encodes a complete set of tRNA needed for mitochondrial translation.
Our studies of mt-genomes in non-bilaterian animals revealed that loss of mt-tRNA genes and import of nuclear tRNA are common in these taxa. For example, all representatives of the phyla the Cnidaria and Ctenophora retained at most two mt-tRNA genes, while the number of tRNA genes in Porifera varies between 2 and 27.
Recently, we characterized a group of sponges where the number of mtDNA-encoded tRNA varies substantially among closely related species, necessitating import of various sets of cy-tRNA into mitochondria.
We plan to harness the observed variation in mt-tRNA gene content and phylogeny-based analysis of selective coefficients to search for candidate proteins involved tRNA import.
We will then search for the homologs of these proteins in bilaterian animals and humans.
If such homologs exist, they can be potentially modified in the model species to test tRNA import as a potential treatment for mitochondrial diseases.
In addition to its potential medical applications, this analysis is expected to reveal the influence of mt-tRNA replacement on the evolution of mtDNA-encoded proteins and help to explain why at least some tRNA genes are retained in mtDNA of most eukaryotes. For this seed grant we are assembling genomic resources for selected species of interest and test various methodologies to be used for mitochondrial extraction and tRNA sequences in sponges. -->
